docs/system/container-metrics.md

```diff
@@ -42,10 +46,74 @@ This metric is [opt-in][MetricOptIn].
 |---|---|---|---|---|---|
-| [`cpu.mode`](/docs/attributes-registry/cpu.md) | string | The CPU mode for this data point. A container's CPU metric SHOULD be characterized _either_ by data points with no `mode` labels, _or only_ data points with `mode` labels. [1] | `user`; `system` | `Opt-In` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`cpu.mode`](/docs/attributes-registry/cpu.md) | string | The CPU mode for this data point. A container's CPU metric SHOULD be characterized _either_ by data points with no `mode` labels, _or only_ data points with `mode` labels. [1] | `user`; `system` | `Conditionally Required` [2] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1]:** Following states SHOULD be used: `user`, `system`, `kernel`
+
+**[2]:** Required if mode is available, i.e. metrics coming from the Docker Stats API.
+
+
+
+`cpu.mode` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `idle` | idle | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `interrupt` | interrupt | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `iowait` | iowait | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `kernel` | kernel | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `nice` | nice | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `steal` | steal | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `system` | system | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `user` | user | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `container.cpu.usage`
+
+This metric is [opt-in][MetricOptIn].
+
+<!-- semconv metric.container.cpu.usage(metric_table) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `container.cpu.usage` | Gauge | `{cpu}` | Container's CPU usage, measured in cpus. Range from 0 to the number of allocatable CPUs [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+
+**[1]:** CPU usage of the specific container on all available CPU cores, averaged over the sample window
+
+
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+<!-- semconv metric.container.cpu.time(full) -->
```

Comment:
This is still wrong, as @lquerel spotted. @ChrsMark curious now, did you add this manually or was it produced by the markdown generation? We aren't supposed to edit these markdown files manually anymore so maybe that was it? 🤔

Reply:
Good catch, I didn't notice that line. It was automatically generated by the tooling. I just changed the `requirement_level` of the `container.cpu.time` metric.
I see, we need to block this until the tooling is fixed.

Reply:
Credits to @lquerel good eyes 😅

Reply:
wait, why are we not supposed to edit these files? It's not the registry, so we added `container.cpu.time` and the fix would be

```suggestion
<!-- semconv metric.container.cpu.usage(full) -->
```

what am I missing?

Reply:
no worries! It seems everything is fixed now.